src/main/java/org/fundacionjala/trello/pages/login/LoginPage.java

```diff
@@ -0,0 +1,57 @@
+package org.fundacionjala.trello.pages.login;
+
+import org.fundacionjala.trello.config.Environment;
+import org.fundacionjala.trello.pages.home.BoardsPage;
+import org.fundacionjala.trello.pages.trello.PageObject;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public final class LoginPage extends PageObject {
+
+    private static final String ENDPOINT = "/login";
```

Comment:
I think this is not the endpoint.
They are the same, but this is a different context.

Reply:
Ok, In this case we can use the "URI" name, because it is no a url complete.